### DISCUSSION TITLE
联通云盘目前无法新建文件夹，这是一个普遍问题

### DISCUSSION THREAD

**Discussion by scituperate** [AlistGo/alist#4881] [General]
联通云盘目前无法新建文件夹，这是一个普遍问题，请给予修复，谢谢
<img width="1222" alt="image" src="https://github.com/alist-org/alist/assets/118343667/f339d46d-141a-4973-b9e6-af1c03ae578c">

**Comment by gracehuahua** [comment]
是的，我也发现这个问题了，新建文件夹会出错
